umap/static/umap/js/umap.layer.js

```diff
@@ -939,11 +939,11 @@ U.DataLayer = L.Evented.extend({
     if (idx !== -1) this._propertiesIndex.splice(idx, 1)
   },
 
-  addData: function (geojson) {
+  addData: function (geojson, sync) {
```

Comment:
Not saying to do it in this PR, but this rings a bell to make sync order out of the internal object process.
In very short, the one who calls `createDataLayer` would also create sync after, or not.

Reply:
Correct, that's something to solve at some point. 👍